Sources/SwiftlyCore/Platform.swift

```diff
@@ -106,6 +143,60 @@ extension Platform {
             throw Error(message: "\(args.first!) exited with non-zero status: \(process.terminationStatus)")
         }
     }
+
+    public func runProgramOutput(_ program: String, _ args: String...) async throws -> String? {
+        let process = Process()
+        process.executableURL = URL(fileURLWithPath: "/usr/bin/env")
+        process.arguments = [program] + args
+
+        let outPipe = Pipe()
+        process.standardInput = FileHandle.nullDevice
+        process.standardError = FileHandle.nullDevice
+        process.standardOutput = outPipe
+
+        try process.run()
+        // Attach this process to our process group so that Ctrl-C and other signals work
+        let pgid = tcgetpgrp(STDOUT_FILENO)
```

Comment:
Can we add the same process group support to `runProgram`

Reply:
I've added this functionality to the runProgram too.